src/main/java/org/openrewrite/java/migrate/AddScopeToInjectedClass.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate;
+
+import lombok.Data;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.ScanningRecipe;
+import org.openrewrite.Tree;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.AnnotationMatcher;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.internal.lang.Nullable;
+import org.openrewrite.java.tree.*;
+
+import java.util.*;
+
+public class AddScopeToInjectedClass extends ScanningRecipe<AddScopeToInjectedClass.Accumulator> {
+    private static final Collection<String> typesPromptingScopeAddition = Arrays.asList("javax.inject.Inject");
```

Comment:
Did you have plans to expand this list now, or in the future? And should that remain in code, or in an Option?

Not too familiar with `javax.inject.Inject` myself, so hoping you'll bear with me asking silly questions once in a while.

Reply:
at some later date the recipe could be parameterized and storing the search target like this would let you search for more than one annotation